sync/work/src/main/java/com/google/samples/apps/nowinandroid/sync/workers/SyncWorker.kt

```diff
@@ -96,4 +102,32 @@ class SyncWorker @AssistedInject constructor(
             .setInputData(SyncWorker::class.delegatedData())
             .build()
     }
+
+    private suspend fun populateDataInFtsTables() {
+        // Populate Data in Topics FTS Table
+        topicRepository.populateDataInTopicsFtsTable(
+            topicRepository.getTopics()
+                .flatMapConcat { it.asFlow() }.toList()
```

Comment:
getTopics() returns a infinity Flow.
So toList() will  not complete.

```suggestion
            topicRepository.getTopics()
                .first()
```